content/sensu-core/1.6/faq.md

```diff
@@ -188,6 +188,20 @@ See the below table for a listing of services, ports, and protocols Sensu uses.
 | Sensu Client Socket               | TCP      | 3030 |
 | Uchiwa/Sensu Enterprise Dashboard | TCP      | 3000 |
 
+> What configuration files do I need for Sensu to function?
+
+See the table below for the location of the respective files needed inside of `/etc/sensu/conf.d`:
+
+| Filename      | Client | Server |
+|---------------|--------|--------|
+| api.json      |        | ✅       |
+| client.json   | ✅       | ✅       |
+| config.json (see note)  |        |        |
+| rabbitmq.json | ✅       |        |
+| redis.json    |        | ✅       |

```

Comment:
Ah! Yep. I neglected dashboard since that doesn't really get brought up often, but you're right. That should be in there. Transport.json slipped my mind.